infrastructure/modules/xc3/output.tf

```diff
@@ -32,6 +32,6 @@ output "private_ip" {
 }
 
 output "load_balancer_dns" {
```

Comment:
```suggestion
output "xc3_url" {
```

Reply:
@yaseenzafar123 please change it everywhere

Reply:
Added in https://github.com/XgridInc/xc3/commit/44dba3bb1d9fa24fab026017c0f60e673cf9c1ef